src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceGetTaskTest.java

```diff
@@ -154,120 +152,6 @@ void should_get_a_task_to_with_challenged_grant_type() {
         });
     }
 
```

Comment:
These test are focused on edge cases to ensure we elegantly return empty task. Are these permutations really duplicates or duplicated?